server.go

```diff
@@ -297,78 +294,34 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 
 		// configure Datadog as sink
 		if ret.DDTraceAddress != "" {
-			ret.tracerSinks = append(ret.tracerSinks, tracerSink{
-				name:   "Datadog",
-				tracer: nil,
-				flush:  flushSpansDatadog,
-			})
-			log.Info("Configured Datadog trace sink")
-		}
 
-		// configure Lightstep as Sink
-		if ret.traceLightstepAccessToken != "" {
-			var host *url.URL
-			host, err = url.Parse(conf.TraceLightstepCollectorHost)
+			var ddSink spanSink
```

Comment:
Gotya, I wasn't aware you could explicitly return but feel dumb for not realizing.

Agreed on named return. Maybe that's a 2.0 fix? :)